app/schemas.py

```diff
@@ -1,17 +1,35 @@
+from enum import Enum
+
 from fastapi import HTTPException, status
 from pydantic import BaseModel, validator
 
+from app.settings import config
+
 
 class ChoiceModel(BaseModel):
-    choice: str
+    your_choice: str
+
+
+class ChoiceResponseModel(BaseModel):
+    your_choice: str
+    pc_choice: str
+    message: str
+
+
+class GameResult(Enum):
+    WIN = "You Win"
+    LOSS = "You Lose"
+    TIE = "Tie"
```

Comment:
Все-таки давай заменим здесь на auto(), представим, что этот GameResult нам удобно хранить в БД в виде int-а 
auto импортируй из enum
ну и соответственно в get_message_from_game_result
```
GAME_RESULT_TO_MSG = {GameResult.WIN: "You win", ...}
```
    
```suggestion
class GameResult(Enum):
    WIN = auto()  # 1
    LOSS = auto()
    TIE = auto()
```